docs/patterns/Auth/connected_account.mdx

```diff
@@ -1,260 +1,167 @@
 <CodeGroup>
-```python Python - New Connection
-
-# This is the URL that the user will be redirected to after completing the authentication process
-redirect_url = "https://yourwebsite.com/connection/callback"
+```python Python
+from composio import ComposioToolSet, App 
+toolset = ComposioToolSet()
 
-entity_id = "Jessica" # This is the unique identifier for the user
+# URL that the user will be redirected to after completing the authentication process
+redirect_url = "https://yourwebsite.com/connection/success"
 
-# Collect the parameters from the user
-# Here we are assuming that the user has provided the parameters
-# In a real-world scenario, you would collect these parameters from the user via your app's UI.
-collected_params = {
-    # "api_key": "1234567890" # This is an example, actual values will be different
-}
+# Unique identifier for the user for whom the connection is being created
+entity_id = "Jessica" 
 
-# Initiate the connection
-connection_request = toolset.initiate_connection(connected_account_params=collected_params,entity_id=entity_id,app=App.LINEAR,redirect_url=redirect_url)
+# Initiate new connection (You can use integration_id instead of app)
+connection_request = toolset.initiate_connection(redirect_url=redirect_url, entity_id=entity_id, app=App.GMAIL)
 
-# If the connection method is OAuth, redirect the user to the URL below
+# Redirect the user to the URL below
 print(connection_request.redirectUrl)
 
-# Check the status of the connection
+# Check the status of the connection (it can be initiated, active, or failed)
 print(connection_request.connectionStatus)
 ```
 
-```javascript Javascript - New Connection
-const redirectUrl = "https://yourwebsite.com/connection/callback"
+```javascript Javascript
+import { OpenAIToolSet } from "composio-core";
+const toolset = new OpenAIToolSet();
 
-const entityId = "Jessica" # This is the unique identifier for the user
+// URL that the user will be redirected to after completing the authentication process
+const redirectURL = "https://yourwebsite.com/connection/success"
 
-// Collect the parameters from the user
-// Here we are assuming that the user has provided the parameters
-// In a real-world scenario, you would collect these parameters from the user via your app's UI.
-const collectedParams = {
-    // "api_key": "1234567890" // This is an example, actual values will be different
-}
+// Unique identifier for the user for whom the connection is being created
+const entityId = "Jessica" 
 
+// Initiate new connection (You can use integration_id instead of app)
 const connectionRequest = await toolset.client.connectedAccounts.initiate({
-    data: {
-        ...collectedParams
-    },
-    userUuid: entityId,
-    integrationId: expectedInputFields.integrationId, 
-    redirectUri: redirectUrl
+  appName: "gmail",
+  redirect_url: redirectURL,
+  entityId: entityId,
+  authMode: "OAUTH2",
+  authConfig: {},
```

Comment:
Consider adding documentation about what values can go into `authConfig`. Currently it's shown as an empty object, but users might need to know what configuration options are available here.